content/client-server-api/modules/content_repo.md

```diff
@@ -19,6 +19,12 @@ When serving content, the server SHOULD provide a
 `Content-Security-Policy` header. The recommended policy is
 `sandbox; default-src 'none'; script-src 'none'; plugin-types application/pdf; style-src 'unsafe-inline'; object-src 'self';`.
 
+{{% added-in v="1.4" %}}
+
+The server SHOULD additionally provide `Cross-Origin-Resource-Policy: cross-origin`
```

Comment:
I thought we weren't doing RFC2119 keywords?

Reply:
Since when? We've always used SHOULD and MUST, just not MAY. It's in the API standards section...

Reply:
:shrug: fair enough. I thought at some point we'd decided not to use it as it's not familiar for most readers.

>  It's in the API standards section...

I don't see this.

Reply:
I swear it was there. Somewhere we describe that we use MUST and SHOULD (which have the same meaning as must and should), but not MAY and such. 

If we don't have this written down somewhere, my whole life has been a lie 😭

Reply:
I'd search github for "MUST", but that isn't going to go well :-S

Reply:
even searching the repo for "MUST" (case sensitive) yields a ton of results, so I'm inclined to say that even though it's not written down, we follow the rule:

<details>
<summary>176 matches over 53 files (tall image)</summary>

![image](https://user-images.githubusercontent.com/1190097/183540361-30cdead4-28b4-465c-a826-b406cd217bc5.png)

</details>